.github/workflows/split_unused_packages.yaml

```diff
@@ -0,0 +1,63 @@
+name: Split Unused Packages
+
+on:
+    pull_request: null
+
+jobs:
+    split_unused_packages:
+        runs-on: ubuntu-latest
+        strategy:
+            fail-fast: false
+            matrix:
+                package_name:
+                    - autowire-array-parameter
+                    - smart-file-system
+                    - package-builder
+                    - easy-coding-standard
+                    - easy-coding-standard-tester
+                    - composer-json-manipulator
+                    - coding-standard
+                    - changelog-linker
+                    - monorepo-builder
+                    - flex-loader
+                    - autodiscovery
+                    - set-config-resolver
+                    - console-color-diff
+                    - easy-hydrator
+                    - easy-testing
+                    - phpstan-extensions
+                    - symfony-static-dumper
+                    - skipper
+
+        name: Unused Packages of ${{ matrix.package_name }}
+
+        steps:
+            -   uses: actions/checkout@v2
+
+            -   uses: shivammathur/setup-php@v2
+                with:
+                    php-version: 7.3
+                    coverage: none
+                    tools: composer:v2
+
+            -   run: composer install --no-progress --ansi
+
+            -   run: packages/monorepo-builder/bin/monorepo-builder localize-composer-paths packages/${{ matrix.package_name }}/composer.json --ansi
+
+            -
+                working-directory: packages/${{ matrix.package_name }}
+                run: composer update --no-progress --ansi
+                env:
+                    # see https://github.com/composer/composer/issues/9368#issuecomment-718112361
+                    COMPOSER_ROOT_VERSION: "dev-master"
+
+            -
+                working-directory: packages/${{ matrix.package_name }}
+                run: composer require icanhazstring/composer-unused --dev --ansi
+                env:
+                    # see https://github.com/composer/composer/issues/9368#issuecomment-718112361
+                    COMPOSER_ROOT_VERSION: "dev-master"
+
+            -
+                working-directory: packages/${{ matrix.package_name }}
+                run: composer unused --ansi
```

Comment:
@JanMikes Hi Honza, we called about this package last week. The one that warns about unused composer depenencies in local packages.

I tested it today and it works :tada:  :) 
Even on split packages. Here I can remove the `symplify/set-config-resolver`:

![image](https://user-images.githubusercontent.com/924196/97873094-6a4fcc80-1d17-11eb-81d2-9368d9433b7e.png)

Reply:
Another useless dependency found:

![image](https://user-images.githubusercontent.com/924196/97873286-b26eef00-1d17-11eb-89bf-2440530ad6e7.png)